src/main/groovy/com/blackducksoftware/integration/hub/detect/hub/HubServiceWrapper.groovy

```diff
@@ -88,24 +88,13 @@ class HubServiceWrapper {
     }
 
     public boolean testHubConnection(boolean detailedLog) {
-        logger.info("Attempting connection to the Hub")
         try {
-            IntLogger slf4jIntLogger;
             if (detailedLog) {
-                slf4jIntLogger = new Slf4jIntLogger(logger);
+                assertHubConnection(new Slf4jIntLogger(logger))
             } else {
-                slf4jIntLogger = new SilentLogger();
+                assertHubConnection()
             }
-            HubServerConfig hubServerConfig = createHubServerConfig(slf4jIntLogger);
-
-            final RestConnection connection = hubServerConfig.createRestConnection(slf4jIntLogger)
-            connection.connect()
-            logger.info("Connection to the Hub was successful")
             return true;
-        } catch (IllegalStateException e) {
```

Comment:
I believe removing this and not modifying any other logs will be fine as the exception message will say that there's an issue with your cert. That's enough of a log imo